lib/v3/error_response.rb

```diff
@@ -0,0 +1,10 @@
+class ErrorResponse
+  attr_accessor :error_identifier, :default_message
+
+  ERRORS = [:login_merge_newSchoolNameMissing].freeze
+
+  def initialize(error_identifier, default_message = nil)
```

Comment:
Maybe specify in docs that `default_message` is sent back in the error response JSON? Or that might just be obvious